py/picca/delta_extraction/least_squares/least_squares_cont_model.py

```diff
@@ -57,6 +57,8 @@ def __init__(self,
         else:
             self.weights_kwargs = weights_kwargs
 
+        self.ndata = None
```

Comment:
ndata: int
number of datapoints used in the fit

I think this should be added to the docstring.

Reply:
addressed in commit